scipy/signal/ltisys.py

```diff
@@ -275,168 +279,144 @@ class lti(object):
 
     Parameters
     ----------
-    args : arguments
+    system : lti system arguments
         The `lti` class can be instantiated with either 2, 3 or 4 arguments.
-        The following gives the number of elements in the tuple and the
-        interpretation:
+        The following gives the number of arguments and the corresponding
+        subclass that is created:
 
-            * 2: (numerator, denominator)
-            * 3: (zeros, poles, gain)
-            * 4: (A, B, C, D)
+            * 2: TransferFunction:  (numerator, denominator)
+            * 3: ZerosPolesGain: (zeros, poles, gain)
+            * 4: StateSpace:  (A, B, C, D)
 
         Each argument can be an array or sequence.
 
     Notes
     -----
-    `lti` instances have all types of representations available; for example
-    after creating an instance s with ``(zeros, poles, gain)`` the transfer
-    function representation (numerator, denominator) can be accessed as
-    ``s.num`` and ``s.den``.
+    `lti` instances do not exist directly. Instead, `lti` creates an instance
+    of one of its subclasses: StateSpace, TransferFunction or ZerosPolesGain.
```

Comment:
I think StateSapce TransferFunction and ZerosPolesGain should all have single-backticks to get proper refs